.travis.yml

```diff
@@ -30,7 +30,7 @@ install:
 before_script:
 - free -m
 - sleep 3
-- if [[ "$DB" == "mysql" ]]; then (mkdir lib; cd lib; curl -Oq http://central.maven.org/maven2/mysql/mysql-connector-java/5.1.39/mysql-connector-java-5.1.39.jar );
+- if [[ "$DB" == "mysql" ]]; then (mkdir lib; cd lib; curl -Oq https://central.maven.org/maven2/mysql/mysql-connector-java/5.1.39/mysql-connector-java-5.1.39.jar );
```

Comment:
this should be repo1.maven.org instead of central.maven.org because of this: SSL: no alternative certificate subject name matches target host name 'central.maven.org'